charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-plugin-controller.yaml

```diff
@@ -170,6 +170,7 @@ spec:
         - --kubeconfig=/var/run/secrets/gardener.cloud/shoot/generic-kubeconfig/kubeconfig
         - "--leader-election=true"
         - "--leader-election-namespace=kube-system"
+        - --handle-volume-inuse-error=false
```

Comment:
This flag can be set only if online expansion is supported by `alibaba-cloud-csi-driver`, can you confirm if online expansion is supported by `alibaba-cloud-csi-driver`?

Reply:
from code https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver/blob/release-v1.1.6/pkg/disk/controllerserver.go#L1024-L1082, expansion is supported